src/resources/resources.css

```diff
@@ -68,7 +69,7 @@
 
 .resources-content {
 	color: white;
-	padding: 3rem 12% 2rem 12%;
+	padding: 2rem 3rem 2rem 3rem;
```

Comment:
if top & bottom padding are same and same for left & right. you can keep it short like 
`padding: 2rem(top-bottom) 3rem(left-right)`
The horizontal padding 3rem is too wide in modile mod, so use percentage instead of 3rem (recommend 5%)

Reply:
Using the (top-bottom) thing doesn't seem to work.

Reply:
It works well for me. Maybe some other css container affect it. I recommend inspect and change attributes right on dev mode. If it takes more time, then skip it.

Reply:
Okay it works if I do only (top-bottom), but if I do (top-bottom) and (left-right) on the same line, it breaks. I'm just going to stick to the standard way anyway since one of the two aren't the same.
<img width="643" alt="Screen Shot 2022-08-16 at 11 03 42 PM" src="https://user-images.githubusercontent.com/36748732/185046203-8ac9e6bc-ff23-4a65-89bc-7ca0d01cb0fc.png">
.

Reply:
(top-bottom) is for explanation, you don't need to add it to your code :)

Reply:
OHHH I see.